src/MoBi.Core/Commands/ChangePathAndValueEntityNameCommand.cs

```diff
@@ -22,7 +23,7 @@ protected ChangePathAndValueEntityNameCommand(TBuildingBlock buildingBlock, Obje
          : base(buildingBlock)
       {
          _newValue = newValue;
-         _originalEntity = buildingBlock[path];
+         _originalEntity = buildingBlock.Single(x => Equals(x.Path, path));
```

Comment:
Sometimes we don't get to use the cache lookup which was nice.

Reply:
I don't get this change here. Why can't it behave the same as before?
Plus not sure if this is equivalent. a cache may return null if this is not found. This will throw. 
And Maybe this is an actual new interface. ILookupBuildingBlock or sthg. We cannot lose suddenly access by path. Or am I missing the problem?

Reply:
I could have used singleOrDefault instead. But let me see about this lookup by path for both types

Reply:
Sure. Then at least it's equivalent. But I think we are loosing some good UX for development